### PR TITLE
Update report by images

### PR DESCRIPTION
**Pull request by mpelekh**

### CODE REVIEW

**Comment by bheptinh** [comment]
probably we should rename `image-service` to `photo-service`

**Comment by mpelekh** [comment]
> probably we should rename `image-service` to `photo-service`

done

File: photo-service/src/queueProducer/rabbitmqProducer.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import {Connection, Channel, connect} from "amqplib";
+import IQueueProducer from "./IQueueProducer";
+import IMessageToUploadPhotos from "./IMessageToUploadPhotos";
+
+export default class RabbitmqProducer implements IQueueProducer {
+    private connection: Connection | any;
```
Comment (zhunik) [inline photo-service/src/queueProducer/rabbitmqProducer.ts]: Can we somehow avoid explicit `any` usage?

File: report-service/queueConsumer/rabbitmqConsumer.ts [hunk 2]
```diff
@@ -1,24 +1,36 @@
...
-        const channel = await this.connection.createChannel();
+    constructor(private url: string) {}
 
-        await channel.assertQueue(queue, {durable: true});
-        await channel.prefetch(1);
-        await channel.consume(queue, async (message: ConsumeMessage | null) => {
-            const messageToUploadImages = message && JSON.parse(message.content.toString());
+    async init() {
+        if (this.initialized) {
+            throw new Error("The RabbitmqConsumer instance already initialised");
```
[the remark this answers is not in the document]
Comment (mpelekh) [reply report-service/queueConsumer/rabbitmqConsumer.ts]: Custom errors contain the HTTP status code which will be redundant for the current case.

File: photo-service/src/queueProducer/rabbitmqProducer.ts (hunk 1 above)
Comment (mpelekh) [reply photo-service/src/queueProducer/rabbitmqProducer.ts]: Makes sense. Fixed.

File: report-service/queueConsumer/rabbitmqConsumer.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (bheptinh) [reply report-service/queueConsumer/rabbitmqConsumer.ts]: Makes sence

**Pull request merged**